skrub/_reporting/tests/test_table_report.py

```diff
@@ -123,6 +127,42 @@ def test_duration(df_module):
     assert re.search(r"2(\.0)?\s+days", TableReport(df).html())
 
 
+@pytest.mark.parametrize("filename_type", ["str", "Path", "file_object"])
```

Comment:
if we want to be super exhaustive maybe we could test files both in text and binary modes?